app/views/checkout/payment.scala.html

```diff
@@ -32,9 +32,7 @@
     bodyClasses = List("is-wide"),
     touchpointBackendResolutionOpt = Some(touchpointBackendResolution)
 ) {
-<main class="page-container gs-container"
-      data-tracking-prop17="GuardianDigiPack:Name and address"
-      data-tracking-products="Subscriptions and Membership;GUARDIAN_DIGIPACK;@{selectedProduct.frequency.numberOfMonths};@{selectedProduct.price};scOpen">
+<main class="page-container gs-container" data-no-page-load-tracking >
```

Comment:
Could we define whether we enable this in the global config too? Maybe defined in the controller?